tests/integration/targets/mso_schema_site_bd_subnet/tasks/main.yml

```diff
@@ -575,7 +642,7 @@
 - name: Verify non_existing_site_template_association
   assert:
     that:
-    - non_existing_site_template_association.msg == "Provided site/template 'ansible_test-Template4' does not exist."
+    - non_existing_site_template_association.msg.startswith("Provided condition(s) ['siteId==")
```

Comment:
This error message is not really user friendly. Why changing it from what it was before?

Reply:
changed because I tried to create a generic message from within the schema class, we can change this message structure

Reply:
changed as discussed